src/lib/dialog/dialog.ts

```diff
@@ -96,6 +99,7 @@ export class MatDialog {
       private _injector: Injector,
       @Optional() location: Location,
       @Inject(MAT_DIALOG_SCROLL_STRATEGY) private _scrollStrategy,
+      @Optional() @Inject(MAT_DIALOG_GLOBAL_OPTIONS) private _options,
```

Comment:
Instead of having this be an optional token, you can clean it up by having the `MatDialogModule` provide the token with a default value:

```
@NgModule({
  providers: [
    {provide: MAT_DIALOG_GLOBAL_OPTIONS, useValue: new MatDialogConfig()}
  ]
})
export class MatDialogModule {
}
```

Doing this also avoids having to add another argument to `_applyConfigDefaults` below.